repo_files/documentations/bar_chart.md

```diff
@@ -158,4 +159,4 @@ enum values {`start`, `end`, `center`, `spaceEvenly`, `spaceAround`, `spaceBetwe
 
 
 ##### Gist - Toggleable Tooltip ([Source Code](https://gist.github.com/imaNNeoFighT/bce3f0169ff3fd6c3f137cdeb5005c0e))
-https://user-images.githubusercontent.com/7009300/156784816-53f95dd9-f387-4600-8a92-d05b1aeea3da.mov
```

Comment:
This is a redundant change. Can you please revert it?

Reply:
Hello @imaNNeoFighT  ! 
I'm not sure what happened here as the two lines are the same 🤔
I don't know how can I revert it ? 😖

Reply:
You can edit your latest commit using `--amend` flag. Like the below command:
`git commit --amend --no-edit` (`--no-edit` is for when you don't want to edit the message, you just need to add your staged changes to the latest commit without changing the commit message).

Also, you can edit the previous commits (before the latest commit) using `interactive rebase` which is an advanced topic. You can find more about it if you google it.